src/components/ClaimErrorAlert/index.tsx

```diff
@@ -115,7 +120,18 @@ export const ClaimErrorAlert = ({
     )
   }
 
-  if (String(error).includes('EXCEEDED_CLAIM')) {
+  if (claimsExceeded) {
+    let exceededClaimMessage = t(
+      'quest.multipleClaimsDetected.message',
+      `You've already claimed this quest the max number of times. Please note that HyperPlay doesn't decide eligibility for this type of quest. If this seems wrong, please open a support ticket using the link below.`
+    )
+    if (gameName && maxNumOfClaims !== undefined && maxNumOfClaims !== null) {
+      exceededClaimMessage = t(
+        'quest.multipleClaimsDetected.messageWithVariables',
+        `You've already claimed this quest with {{maxNumOfClaims}} wallet(s).\n{{gameName}} allows a max of {{maxNumOfClaims}} claims per user to keep things fair.\nPlease note that HyperPlay doesn't decide eligibility for leaderboard quests — that's handled by {{gameName}}. If this seems wrong, please open a support ticket using the link below.`,
+        { gameName, maxNumOfClaims }
```

Comment:
being careful here with a fallback. also makes this package easier to adopt in the store